lib/px/utils/pxlogin_credentials.lua

```diff
@@ -107,7 +107,10 @@ function M.load(px_config)
     function _M.creds_extract_from_body_formdata(ci)
         -- maximal POST field size to read
         local chunk_size = 4096
```

Comment:
I see those number in the upload file as well
Why on both?

Reply:
If they are not aligned is it still fine?

Reply:
There is only 1 "chunk_size" variable, in "upload.lua" it's a default constant.

Reply:
I meant those two:
local CHUNK_SIZE = 4096
local MAX_LINE_SIZE = 512

if those are default, so why passing default from our side?

Reply:
yes, these are default values.
the reason I'm using these values:
1) to explicitly see which values are set (as the default values could change in the future)
2) (more importantly) we need to set the 3rd argument of that function (which is `restore_body_buffer`, setting it to `true`)